README.md

```diff
@@ -8,6 +8,27 @@ Module to:
 * automatically generate OpenAPI (Swagger) specification for existing ExpressJS 4.x REST API applications;
 * provide Swagger UI basing on generated specification.
 
+## Additions to your package.json
+
+add your service base url path to package.json
+```angular2
```

Comment:
Why is it `angular2` here and below? Isn't it server side package.json? Would `javascript` be better?

Reply:
i updated it to json